spec/ruby/language/fixtures/defined.rb

```diff
@@ -94,6 +94,11 @@ def global_variable_defined
       defined? $defined_specs_global_variable_defined
     end
 
+    def global_variable_defined_as_nil
+      $defined_specs_global_variable_defined_as_nil = nil
+      defined? $defined_specs_global_variable_defined_as_nil
+    end
```

Comment:
Where is the spec? :wink:

Reply:
Oops. Had to do an unscheduled reboot, and clearly lost two saves.